build.rs

```diff
@@ -1,41 +1,67 @@
 extern crate cmake;
+extern crate curl;
 
-use std::env;
+use curl::easy::Easy;
 use std::fs::File;
 use std::io::{Read, Write};
 use std::path::Path;
 use std::process::Command;
 use std::thread;
 
 /// Build from https://wasm-stat.us that we are known to work with.
-const WASM_BUILD: &'static str = "9901";
+#[cfg(target_os="linux")]
+mod config {
+    pub const DOWLOAD_WASM: bool = true;
+    pub const WASM_BUILD: &'static str = "14533";
+    pub const OS_NAME: &'static str = "linux";
+}
+
+#[cfg(target_os="macos")]
+mod config {
+    pub const DOWLOAD_WASM: bool = true;
+    pub const WASM_BUILD: &'static str = "2670";
+    pub const OS_NAME: &'static str = "mac";
+}
+
+#[cfg(not(any(target_os="linux", target_os="macos")))]
+mod config {
+    pub const DOWLOAD_WASM: bool = false;
+    pub const WASM_BUILD: &'static str = "";
+    pub const OS_NAME: &'static str = "";
```

Comment:
FWIW the windows bot is now uploading a build as well. It doesn't run all of the tests yet but I think the binary is working. Obviously don't need to update this PR for that necessarily but FYI.